homeassistant/components/ephember/climate.py

```diff
@@ -89,8 +89,10 @@ def __init__(self, ember, zone):
         self._ember = ember
         self._zone_name = zone_name(zone)
         self._zone = zone
-        self._hot_water = zone_is_hot_water(zone)
-
+        self._hot_water = False
+        # self._immersion = False
+        self._immersion = zone['deviceType'] == 4
```

Comment:
Please don't keep adding unexplained magic numbers, either import the correct enum type from the library or add a comment explaining what 4 is.

Reply:
Not sure how to do enum type. I know devicetype of 4 is returning a hot water type thermostat so hard coded any device types to that.